spec/system/managing_topics_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'managing topics' do
+  before do
+    FactoryBot.create(:topic, name: 'What is your favourite colour')
+  end
+
+  scenario 'when viewing' do
+    visit('/topics')
+    expect(page).to have_content('Topics')
+    expect(page).to have_content('What is your favourite colour')
+  end
+
+  scenario 'can navigate to new topic page' do
```

Comment:
System/feature specs should test a single user operation. In this case, the user is trying to _create a topic_, so roll this spec into the the _'when creating a topic'_ spec.

Reply:
We can remove this spec now right?